main.go

```diff
@@ -32,6 +32,15 @@ func logrus_init(d bool) {
 	}
 }
 
+func validate(n string) {
+	r := regexp.MustCompile(`^[0-9]*$|^[0-9]*..[0-9]*$`)
```

Comment:
The regular expression pattern should be as follows.
```
`^[0-9]+$|^[0-9]+..[0-9]+$`
```

If this implementation,  `./qs -n .. ` pattern is not cathed as the validate method.